homeassistant/components/mqtt/cover.py

```diff
@@ -611,12 +611,11 @@ async def async_close_cover_tilt(self, **kwargs):
 
     async def async_set_cover_tilt_position(self, **kwargs):
         """Move the cover tilt to a specific position."""
-        template = self._config.get(CONF_TILT_COMMAND_TEMPLATE)
         tilt = kwargs[ATTR_TILT_POSITION]
         percentage_tilt = tilt
         tilt = self.find_in_range_from_percent(tilt)
         # Handover the tilt after calculated from percent would make it more consistent with receiving templates
-        if template is not None:
+        if self._config.get(CONF_TILT_COMMAND_TEMPLATE) is not None:
```

Comment:
Can we remove this check, it should be OK to render an `MqttCommandTemplate` when its template is `None`?

Reply:
Removed, because the value is not evaluated from the variables, value is passed as well